MissionControl/GUI_threads.py

```diff
@@ -214,28 +219,30 @@ def getLatestSerialInput():
                 values = line.strip().split(',')
 
                 if values[0] == '':
-                    values[0] = str(last_first_value);
+                    values[0] = str(last_first_value)
                 if len(values) < total_sensors:
                     print("not enough data, continuing")
                     continue
                 last_values = values
                 values = [val.strip() for val in values]
+                values = [lowPressureConversion(val) if i != 0 else highPressureConversion(
+                    val) for val, ind in enumerate(values)]
```

Comment:
Sorry, my auto formatting makes it look like there's a lot of changes, but this is the only substantive change in this file.